virtool/users/api.py

```diff
@@ -126,7 +126,7 @@ async def put(self, data: CreateFirstUserRequest) -> Union[r201[User], r400, r40
         """
         db = self.request.app["db"]
 
-        if await db.users.count_documents({}, limit=1):
+        if not await db.users.check_is_empty():
```

Comment:
This is wrong. Please write an API test to make sure this will be caught in the future.